extensions/amp-story/1.0/page-advancement.js

```diff
@@ -565,10 +574,21 @@ export class ManualAdvancement extends AdvancementConfig {
    * @private
    */
   isInStoryPageSideEdge_(event, pageRect) {
-    return (
-      event.clientX <= pageRect.x + PROTECTED_SCREEN_EDGE_PX ||
-      event.clientX >= pageRect.x + pageRect.width - PROTECTED_SCREEN_EDGE_PX
-    );
+    const sideEdgeWidth =
+      pageRect.width * (PROTECTED_SCREEN_EDGE_PERCENT / 100);
+
+    if (sideEdgeWidth > MINIMUM_PROTECTED_SCREEN_EDGE_PX) {
```

Comment:
We can extract this conditional into a new variable so we don't have repeated logic, e.g.:
```
const sideEdgeLimit = Math.max(sideEdgeWidth,  MINIMUM_PROTECTED_SCREEN_EDGE_PX);

return (
        event.clientX <= pageRect.x + sideEdgeLimit ||
        event.clientX >= pageRect.x + pageRect.width - sideEdgeLimit
      );

```